src/Agent.Plugins/BuildServer.cs

```diff
@@ -10,16 +10,18 @@
 
 namespace Agent.Plugins.Drop
 {
-    public class BuildServer
+    // A client wrapper interacting with TFS/Build's Artifact API
+    public class BuildServiceClient
```

Comment:
i am not really interested on class name change, all VSTS client wrapper class in following naming convention `xxxxServer`